apps/services/schemas.py

```diff
@@ -125,6 +125,24 @@ class UserRequestDetailsViewOut(BaseOutSchema):
     documents: List[UserDocumentsSchema]
 
 
+class ReturnUserDocumentIn(BaseInSchema):
```

Comment:
No-no-no, UI don't send any status to us. Delete this useless schema

Reply:
I used it just for validation, so it don't request any status. We request status from SQL
Maybe I should replace this validation from this "InSchema" to handler or another location?